lightning/backend/session.go

```diff
@@ -66,6 +68,7 @@ func newSession(sqlMode mysql.SQLMode) *session {
 	vars.StmtCtx.AllowInvalidDate = sqlMode.HasAllowInvalidDatesMode()
 	vars.StmtCtx.IgnoreZeroInDate = !sqlMode.HasStrictMode() || sqlMode.HasAllowInvalidDatesMode()
 	vars.StmtCtx.TimeZone = vars.Location()
+	vars.SetSystemVar("timestamp", strconv.FormatInt(timestamp, 10))
```

Comment:
What's the usage of this SystemVar, I didn't find usage in lightnong or doc of TiDB

Reply:
It's from MySQL. https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_timestamp